application/actions/organizations.go

```diff
@@ -9,7 +9,7 @@ import (
 
 func convertOrganizationsToAPIType(organizations models.Organizations) ([]api.Organization, error) {
 	output := make([]api.Organization, len(organizations))
-	for i := range organizations {
+	for i := range output {
```

Comment:
I am not sure what difference this makes. I think it was a suggestion by @fillup , right?

Reply:
Phillip said it was easier for him to understand this way, so I thought I would try to make it consistent.